std-bits/base/src/main/java/org/enso/base/Time_Utils.java

```diff
@@ -97,10 +98,38 @@ public static LocalTime time_adjust(LocalTime time, AdjustOp op, Duration durati
     }
   }
 
+  /**
+   * Gets the weekday of the date as a long.
+   * With Monday as 1 through to Sunday as 7.
+   * @param date a datetime
+   * @return the week day as an integer of the date.
+   */
+  public static long day_of_week_localdate(LocalDate date) {
+    return date.getDayOfWeek().getValue();
+  }
+
+  /**
+   * Gets the weekday of the date as a long.
+   * With Monday as 1 through to Sunday as 7.
+   * @param date a datetime
+   * @return the week day as an integer of the date.
+   */
+  public static long day_of_week_zoneddatetime(ZonedDateTime date) {
+    return date.getDayOfWeek().getValue();
+  }
+
+  public static long week_of_year_localdate(LocalDate date) {
+    return date.get(IsoFields.WEEK_OF_WEEK_BASED_YEAR);
+  }
+
   public static long week_of_year_localdate(LocalDate date, Locale locale) {
     return WeekFields.of(locale).weekOfYear().getFrom(date);
   }
 
+  public static long week_of_year_zoneddatetime(ZonedDateTime date) {
+    return date.get(IsoFields.WEEK_OF_WEEK_BASED_YEAR);
+  }
+
   public static long week_of_year_zoneddatetime(ZonedDateTime date, Locale locale) {
     return WeekFields.of(locale).weekOfYear().getFrom(date);
   }
```

Comment:
Do we need these helpers? Cannot we just call these methods from within Enso directly?

Reply:
No because we don't have the java instance anymore with the built-in.

I have moved to a more common function for getting values from Java.

Reply:
Ahhh, ok. I thought that resolution on our builtins would fall back to Java methods if needed. But from user's perspective maybe it is even better if it doesn't.